### PR TITLE
[14.0][IMP] storage_image: Add a wizard to replace related file

### PR DESCRIPTION
**Pull request by mmequignon**
Adds a wizard on the image form to replace the attached file.

### CODE REVIEW

File: storage_file/wizards/replace_file.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    _name = "storage.file.replace"
+    _description = "TODO"
+    
+    file_id = fields.Many2one("storage.file")
+    new_file_id = fields.Many2one("storage.file")
+    data = fields.Binary()
+    file_name = fields.Char()
+
+    @api.onchange("data")
+    def onchange_data(self):
```
Comment (simahawk) [inline storage_file/wizards/replace_file.py]: why on onchange? This is going to cause the creation of tons of orphan files IMO

**Review by simahawk — CHANGES_REQUESTED** [review]
Thanks for the improvement :)

Can you
1. add the wizard to storage_media too
2. drop the onchange approach
3. split commit by module
4. drop the funny image
5. fix pre-commit

please? :smile:

**Comment by mmequignon** [comment]
> Thanks for the improvement :)
> 
> Can you
> 
>     1. add the wizard to storage_media too
> 
>     2. drop the onchange approach
> 
>     3. split commit by module
> 
>     4. drop the funny image
> 
>     5. fix pre-commit
> 
> 
> please? 😄

baaah, you're not funny

**Review by simahawk — COMMENTED** [review]
LG, pls drop the icon. It's automatically generated ;)

File: storage_file/wizards/replace_file.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class StorageFileReplace(models.TransientModel):
+    _name = "storage.file.replace"
+    _description = "TODO"
```
Comment (TDu) [inline storage_file/wizards/replace_file.py]: ?

File: storage_image/tests/test_storage_replace_file.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Camptocamp SA
```
Comment (TDu) [inline storage_image/tests/test_storage_replace_file.py:1]: `__init__.py` file is needed in the directory for the tests to be run on Travis

**Review by TDu — COMMENTED** [review]
Some `tests` directory are missing an `__init__.py` file

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@TDu could you update your review pls?

**Review by TDu — APPROVED** [review]

**Pull request merged**